test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +129,47 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name, sagemaker_session)
+
+def _check_for_cloudwatch_logs(endpoint_name, sagemaker_session):
+    client=sagemaker_session.boto_session.client('logs')
+    log_group_name='/aws/sagemaker/Endpoints/'+endpoint_name
```

Comment:
nit, generally we use f-strings.
```
"/aws/sagemaker/Endpoints/{endpoint_name}"
```
Additionally, it looks like there are a few places where formatting is off. Can you use this formatting tool - https://github.com/psf/black - on this file?

Once you install the package (via pypi), you can run the following command

```
black -l 120 test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py
```

Reply:
Missed this somehow. I just rectified that and then reformatted.